components/Column.tsx

```diff
@@ -1,9 +1,31 @@
+import styled from 'styled-components';
 import { Text, Col } from '@geist-ui/react';
 import { Droppable } from 'react-beautiful-dnd';
 import { Story, Owner, Label, Iteration } from '../redux/types';
 import { useTheme } from '@geist-ui/react';
 import StoryCard from './StoryCard';
 
+const Header = styled(Text)(props => ({
+  color: `${props.color} !important`,
+  textAlign: 'center',
+  textTransform: 'capitalize',
+  fontFamily: 'Georgia, Times New Roman, Times, serif',
+}));
+
+const ColumnContainer = styled(Col)(props => ({
+  borderTop: props.bordertop,
+  backgroundColor: props.backgroundcolor,
+  boxShadow: props.boxshadow,
+  margin: 10,
+  padding: 5,
```

Comment:
Since it looks like we're using 5 as our spacing unit, suggesting to make a `styles` module that exposes a spacing function that takes a step parameter and multiplies the unit * step to yield your desired spacing.
```typescript
// styles/index.ts
const unit = 5;
export const spacing = step => step * unit;
```

Reply:
Awesome! Thanks! 👍🏽